defaults/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
 apache_mirror: http://apache.mirrors.tds.net
-kafka_id: 0
+kafka_hosts: "{{ansible_fqdn}}:9092"
 kafka_version: 0.8.1.1
 skip_install: False
+
+kafka_num_network_threads: 2
+kafka_num_io_threads: 2
+kafka_num_partitions: 2
+kafka_socket_send_buffer_bytes: 1048576
+kafka_socket_receive_buffer_bytes: 1048576
+kafka_socket_request_max_bytes: 104857600
+kafka_log_flush_interval_messages: 10000
+kafka_log_flush_interval_ms: 1000
+kafka_log_retention_hours: 24
+kafka_log_segment_bytes: 536870912
+kafka_log_cleanup_interval_mins: 1
+zookeeper_connection_timeout_ms: 1000000
```

Comment:
I'm not sure I agree with setting all of these. I've seen arguments that you should only set what you need and let the software pick the best values for the rest. This hardcodes values to a certain set and doesn't allow a newer release to change the defaults